spec/services/reporting/agency_and_sp_report_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'csv'
+require 'rails_helper'
+
+RSpec.describe Reporting::AgencyAndSpReport do
+  let(:report_date) do
+    Date.new(2021, 1, 1).in_time_zone('UTC')
+  end
+
+  let(:header_row) do
+    ['', 'Number of apps (SPs)', 'Number of agencies']
+  end
```

Comment:
🤷 clearer assertion failures are one of the reasons we do the `.zip` approach like this:

https://github.com/18F/identity-idp/blob/64cb10fd016f7ecc0b9d2bc277b70ec45fe4507c/spec/lib/reporting/proofing_rate_report_spec.rb#L64-L68

Reply:
🤔 I like that approach, but I think it would be messy to implement here. rspec does surprisingly well with what's here:

```
expected collection contained:  [["", "Number of apps (SPs)", "Number of agencies"], ["Auth", 1, 1], ["IDV", 0, 0]]
actual collection contained:    [["", "Number of apps (SPs)", "Number of agencies"], ["Auth", 0, 1], ["IDV", 0, 0]]
the missing elements were:      [["Auth", 1, 1]]
the extra elements were:        [["Auth", 0, 1]]
```

The `header_row` was less about readable errors, and more about readability of the test code itself.